MSCL/source/mscl/MicroStrain/MIP/MipTypes.h

```diff
@@ -1677,6 +1679,48 @@ namespace mscl
 
     typedef std::vector<DeviceCommPort> CommPortInfo;
 
+    //API Struct: EventTypeInfo
+    //  Information about event trigger or action types
+    struct EventTypeInfo
+    {
+        EventTypeInfo(const uint8 type, const uint8 maxInstances) :
+            type(type),
+            maxInstances(maxInstances)
+        { }
+
+        // Trigger or action type
```

Comment:
Add API Variable full doc comment
Specify the enums that would be used to map the type for each query type:
- Query::TRIGGERS -> EventTriggerConfiguration::Trigger
- Query::ACTIONS -> EventActionConfiguration::Type

Probably should standardize the names of these though. Probably makes sense to change ActionConfig::Type to ::Action because it's not in master yet, but my vote is to switch them both the ::Type - we'll figure it out, nothing to hold up this change for.